tasks/meta.py

```diff
@@ -6,23 +6,17 @@
 
 import os
 import re
-import weakref
 
-import luigi
-from luigi import Task, BoolParameter, Target, Event
+from luigi import Target
 
-from sqlalchemy import (Column, Integer, Text, Boolean, MetaData, Numeric, cast,
-                        create_engine, event, ForeignKey, PrimaryKeyConstraint,
-                        ForeignKeyConstraint, Table, exc, func, UniqueConstraint,
-                        Index)
+from sqlalchemy import (Column, Integer, Text, MetaData, Numeric, cast,
+                        create_engine, event, ForeignKey, ForeignKeyConstraint,
+                        exc, func, UniqueConstraint, Index)
 from sqlalchemy.dialects.postgresql import JSON
 from sqlalchemy.ext.associationproxy import association_proxy
 from sqlalchemy.ext.declarative import declarative_base
-from sqlalchemy.orm import relationship, sessionmaker, composite, backref
```

Comment:
I assume this imports are not longer required

Reply:
We need `relationship`, `sessionmaker` and `backref` ([line 18](https://github.com/CartoDB/bigmetadata/pull/386/files#diff-ec4044a9a28b8936802647b9b5977c8dR18))